tool/tctl/common/resource_command_test.go

```diff
@@ -1683,3 +1687,54 @@ version: v7
 	_, err = runResourceCommand(t, fc, []string{"create", "-f", roleYAMLPath})
 	require.NoError(t, err)
 }
+
+func testCreateServerInfo(t *testing.T, fc *config.FileConfig) {
+	// Ensure that our test server info does not exist
+	_, err := runResourceCommand(t, fc, []string{"get", types.KindServerInfo + "/test-server-info", "--format=json"})
+	require.True(t, trace.IsNotFound(err), "expected test-role to not exist prior to being created")
+
+	const serverInfoYAML = `kind: server_info
```

Comment:
nit: I think this would read a little better if you started the yaml document with `---` - then all the fields are aligned to the left after that:

        const serverInfoYAML = `---
    kind: server_info
    sub_kind: cloud_info
    ...